File: csharp/Svix/Models/EventTypePatch.cs
// this file is @generated
using System.Text.Json.Serialization;

namespace Svix.Models
{
    public class EventTypePatch(
        Object? schemas = null,
        string? groupName = null,
        string? featureFlag = null,
        string? description = null,
        bool? deprecated = null,
        bool? archived = null
    ) : BaseModel
    {
        [JsonPropertyName("archived")]
        public bool? Archived { get; set; } = archived;

        [JsonPropertyName("deprecated")]
        public bool? Deprecated { get; set; } = deprecated;

        [JsonPropertyName("description")]
        public string? Description { get; set; } = description;

        [JsonPropertyName("featureFlag")]
        public string? FeatureFlag { get; set; } = featureFlag;

        [JsonPropertyName("groupName")]
        public string? GroupName { get; set; } = groupName;

        [JsonPropertyName("schemas")]
        public Object? Schemas { get; set; } = schemas;
    }
}

File: kotlin/lib/src/main/kotlin/models/EventTypePatch.kt
// This file is @generated
package com.svix.kotlin.models

import com.svix.kotlin.MaybeUnset
import kotlinx.serialization.Serializable
import kotlinx.serialization.json.JsonObject

@Serializable
data class EventTypePatch(
    val archived: Boolean? = null,
    val deprecated: Boolean? = null,
    val description: String? = null,
    val featureFlag: MaybeUnset<String> = MaybeUnset.Unset,
    val groupName: MaybeUnset<String> = MaybeUnset.Unset,
    val schemas: MaybeUnset<JsonObject> = MaybeUnset.Unset,
)
